Improve serverless custom installation doc (#10965)
DataDog/documentation: content/en/serverless/installation/python.md

{{% /tab %}}
{{% tab "Custom" %}}

> <div class="alert alert-warning">If you are not using any of the recommended serverless development tools, such as the Serverless Framework, we strongly encourage you instrument your serverless applications with the <a href="/?tab=datadogcli">Datadog CLI</a>.</div>

Reviewer: Same comment as in the other file

> <div class="alert alert-info">If you are not using a serverless development tool that we support, such as the Serverless Framework or AWS CDK, we strongly encourage you instrument your serverless applications with the <a href="/?tab=datadogcli">Datadog CLI</a>.</div>

Reviewer: ```suggestion
<div class="alert alert-info">If you are not using a serverless development tool that Datadog supports, such as the Serverless Framework or AWS CDK, we strongly encourage you instrument your serverless applications with the <a href="?tab=datadogcli">Datadog CLI</a>.</div>
```

> <div class="alert alert-warning">If you are not using any of the supported serverless application development tool, such as the serverless framework, we strongly recommend that you set up the Datadog instrumentation using the <a href="/?tab=datadogcli">Datadog CLI</a> instead.</div>

Reviewer 2: ```suggestion
<div class="alert alert-warning">If you are not using any of the recommended serverless development tools, such as the Serverless Framework, we strongly encourage you instrument your serverless applications with the <a href="/?tab=datadogcli">Datadog CLI</a>.</div>
```

### Install the Datadog Lambda library

> The Datadog Lambda Library can be imported either as a layer (recommended) **OR** as a Python package.

Reviewer: ```suggestion
The Datadog Lambda Library can be imported either as a layer (recommended) _or_ as a Python package.
```

> You can either install the Datadog Lambda library as a layer (recommended) **OR** Python package.

Reviewer 2: ```suggestion
The Datadog Lambda Library can be imported either as a layer (recommended) **OR** as a Python package.
```
